src/panda_qt/main_window.cpp

```diff
@@ -139,16 +142,23 @@ void MainWindow::swapEmuBuffer() {
 }
 
 void MainWindow::selectROM() {
-	auto path =
-		QFileDialog::getOpenFileName(this, tr("Select 3DS ROM to load"), "", tr("Nintendo 3DS ROMs (*.3ds *.cci *.cxi *.app *.3dsx *.elf *.axf)"));
+	if (emu->getConfig().getRomsPath() == "") {
+		auto path =
```

Comment:
Both of these paths are identical, there's no point in splitting them, the "" argument should just be feplaced with emu->getConfig().romPath (romsPath should also be called romPath instead of romsPath)